app/kubemci/pkg/gcp/forwardingrule/forwardingrulesyncer.go

```
@@ -123,6 +124,30 @@ func (s *ForwardingRuleSyncer) GetLoadBalancerStatus(lbName string) (*status.Loa
 	return status, nil
 }
 
+func (s *ForwardingRuleSyncer) GetForwardingRules() ([]status.LoadBalancerStatus, error) {
+	var rules *compute.ForwardingRuleList
+	var err error
+	result := []status.LoadBalancerStatus{}
+	// TODO: When we have HTTPS ingresses, check for those as well.
+	if rules, err = s.frp.ListGlobalForwardingRules(); err != nil {
+		fmt.Println("Error getting global forwarding rules:", err)
+		return result, err
+	}
+	// TODO: Should we check for nil rules?
+	glog.V(5).Infof("rules: %+v", rules)
+	for _, item := range rules.Items {
+		if strings.HasPrefix(item.Name, "mci1") {
+			if lbStatus, err := status.FromString(item.Description); err != nil {
+				fmt.Println("Error decoding load balancer status:", err)
```

Shouldnt we return this error as well?